### ISSUE TITLE
ignore .vscode directory 

### ISSUE THREAD

**Issue by AbhimanyuAryan** [github-for-unity/Unity#521]
Hi, I'm using Vscode but .gitignore doesn't exclude `.vscode` directory from project so ends up pushing to remote. Please fix this

**Comment by StanleyGoldman** [comment]
Hey @AbhimanyuAryan thanks for participating and trying out our stuff.
We will definitely add `.vscode` to our ignore file for the future...

But you know you can do this for yourself in the mean time right?

**Comment by thorade** [comment]
If you start adding more general files, you could also add the following:
```
# IDE settings etc
/.vs/
/.vscode/
/.idea/

# Ignore some MS Windows files
desktop.ini
thumbs.db

# Ignores some MacOS files
.DS_Store
._*
```

**Comment by StanleyGoldman** [comment]
Fixed by #748
